src/js/letters/actions/letters.js

```diff
@@ -2,28 +2,28 @@ import Raven from 'raven-js';
 
 import { apiRequest } from '../utils/helpers.jsx';
 import {
+  ADDRESS_TYPES,
   BACKEND_AUTHENTICATION_ERROR,
   BACKEND_SERVICE_ERROR,
-  GET_LETTERS_FAILURE,
-  GET_LETTERS_SUCCESS,
+  GET_ADDRESS_COUNTRIES_SUCCESS,
+  GET_ADDRESS_COUNTRIES_FAILURE,
```

Comment:
As long as we're alphabetizing, `FAILURE` comes before `SUCCESS`.

...how philosophical...

Reply:
hehe you're right :) will fix!